dyad-2/css/blocks.css

```diff
@@ -34,6 +34,23 @@ Description: Used to style Gutenberg Blocks.
 		margin-right: -20%;
 		width: auto;
 	}
+
+	body:not(.has-post-thumbnail) .alignfull .wp-block-group__inner-container .alignwide {
```

Comment:
Do we need similar styles for cover + columns blocks?

Reply:
Answer: Yes. 😄 I'll work on adding that in.

Reply:
Oh shoot! Forgot to change the selector here. `.wp-block-group__inner-container` should be `*[class="__inner-container"]` so any new Blocks with nested styles gets solved here.

Reply:
Ah, ok cool. I can make that change. Columns are structured a little differently, but I'll sort that out too.

Reply:
Fixed in 94ee61e